application-analytics-ui/src/main/resources/Analytics/Code/Macros/VisitsOverview.xml

```diff
@@ -393,7 +393,7 @@
     "analytics.visitsOverview.bounced": ['bounce_count', 'VisitsSummary.get'],
     "analytics.visitsOverview.nb_actions": ['nb_actions_per_visit', 'VisitsSummary.get'],
     "analytics.visitsOverview.maxActionsInOneVisit": ['max_actions', 'VisitsSummary.get'],
-    "analytics.visitsOverview.viewsAndUnique": ['VALUES&amp;VALUES',
+    "analytics.visitsOverview.viewsAndUnique": ['values&amp;values',
```

Comment:
Why you have to send `values&values`?

Reply:
@oanalavinia 
The endpoints for this field return only a single value, and the response format from Matomo is as follows:
```json
{
  "date": "value"
}
Due to this, I have opted to use a placeholder to maintain code consistency and avoid the necessity of handling this case separately.